packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -211,12 +211,24 @@ export const sandboxChanged: AsyncAction<{ id: string }> = withLoadApp<{
 
     actions.internal.setCurrentSandbox(sandbox);
     actions.workspace.openDefaultItem();
+    state.editor.unprocessableEntityError = false;
```

Comment:
I would move this way up to the top of the action, so that every time we start running this logic we "reset" the error. Mr David Xstate would scream if he saw this though. We should ideally have an ENUM of `SandboxError`, being  "notFound" or "unprocessableEntity"... as it can not be both at the same time. 

So if you have time, make the `state.editor.error` to be an object of: `{ message: string, reason: SandboxErrorReason }`. 

But noooo biggie, just if you want to. Call me if you want help 😄